packages/esbuild/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './src/executors/esbuild/schema';
+export type * from './src/executors/esbuild/schema';
```

Comment:
Can you update this to export only options
`EsBuildExecutorOptions` and `NormalizedEsBuildExecutorOptions`

Reply:
Done!